noisemaker/src/Noisey.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import Sound from 'react-sound';
+import { Container, Row, Col, Button } from 'reactstrap';
+
+const sounds = new Audio("bensound-summer.mp3");
+
+
+class Noisey extends Component {
+    constructor(props) {
+        super(props);
```

Comment:
new way of doing this would be to use arrow functions which would bind automatically to the class so you wouldnt have a need for a constructor.